tests/integration/test_relocatable_cores.py

```diff
@@ -53,8 +53,13 @@ def test_single_thread_stack_for_relocated_core(
         # Hack: help libdw find the split debug for ld-musl on Alpine.
         # Otherwise, the test fails even though we do everything right.
         ld_musl_debug = list(Path("/usr/lib/debug/lib/").glob("ld-musl*.debug"))
-        if ld_musl_debug != []:  # pragma: no cover
-            (relocated_bundle / "app" / ".debug").symlink_to("/usr/lib/debug/lib")
+        for debuginfo in ld_musl_debug:  # pragma: no cover
+            (relocated_bundle / "app" / debuginfo.name).symlink_to(debuginfo)
+
+        # Apply the same hack to find the libpython debug info, too.
+        ld_musl_debug = list(Path("/usr/lib/debug/usr/lib/").glob("libpython*.debug"))
```

Comment:
Probably we want to rename this variable

Reply:
Oops, yep.